packages/web-client/app/components/card-pay/layer-two-connect-card/index.hbs

```diff
@@ -72,11 +72,6 @@
         class="layer-two-connect-card__section layer-two-connect-card__section--one"
         data-test-layer-2-connect-prompt
       >
-        {{#if @includeExplanations}}
-          <p class="layer-two-connect-card__explanation">
-            Please install the Card Wallet app on your mobile phone and create/add an account.
-          </p>
-        {{/if}}
```

Comment:
I think this text is for use outside of the workflows. Within workflows, this message precedes the card:

> Once you have installed the app, open the app and add an existing wallet/account or create a
 new wallet/account. Use your account to scan this QR code, which will connect your account with Card Pay.
 
Outside workflows, there is no other copy that mentions the step of creating/adding an account. This copy may still be relevant because of that.

Reply:
Also if we remove this text, we should probably also remove the `@includeExplanations` for the section below.